lib/reducers/create-otp-reducer.js

```diff
@@ -49,6 +51,46 @@ const MAX_RECENT_STORAGE = 5
 
 // TODO: fire planTrip action if default query is complete/error-free
 
+// a list of required keys that must be present in the initial config of the
+// initial state
+const requiredConfigKeys = [
+  'geocoder.type'
```

Comment:
Sorry I didn't think of this before, but should `geocoder.type` actually be a required key? What if someone wants to set up the app without a geocoder defined? I don't think we should throw an error that prevents the site from loading in this case.

Reply:
Hmmm, if it's possible to not have a geocoder, some other things should probably change to make things work properly.